internal/driver/driver.go

```diff
@@ -155,17 +155,17 @@ func applyCommandOverrides(cmd string, outputFormat int, v variables) variables
 	trim := v["trim"].boolValue()
 
 	switch cmd {
-	case "callgrind", "kcachegrind":
-		trim = false
-		v.set("addresses", "t")
 	case "disasm", "weblist":
 		trim = false
-		v.set("addressnoinlines", "t")
+		v.set("addresses", "t")
+		v.set("noinlines", "t")
 	case "peek":
 		trim = false
 	case "list":
-		v.set("nodecount", "0")
+		trim = false
```

Comment:
explanation for this will be helpful for readers

Reply:
Done - added explanatory text above near the `trim` variable declaration.